src/typing/macroContext.ml

```diff
@@ -248,13 +266,13 @@ let make_macro_com_api com p =
 			Interp.exc_string "unsupported"
 		);
 		current_module = (fun() ->
-			Interp.exc_string "unsupported"
+			null_module
```

Comment:
Hmm... this makes some sense, but also seems a bit inconsistent because all the `get_` functions still throw "unsupported".

Reply:
Yeah but they should not be reachable anymore, while that one is being used internally.